examples/plotting/file/pixel_fun.py

```diff
@@ -0,0 +1,37 @@
+#Just for fun, taking a jpg image and plotting it pointillistic style on a bokeh canvas using circle glyphs!
+#Requires PIL!
+import Image
+import numpy
+import random
+from bokeh.plotting import *
+
+
+def int_to_hex_color(v):
+    assert(len(v) == 3)
+    return '#%02x%02x%02x' % (v[0], v[1], v[2])
```

Comment:
"Should" because this particular code path (as used in this example) may actually not work expected.